### ISSUE TITLE
Zubehörstatus RoboVac X8 Hybrid

### ISSUE THREAD

**Issue by MikeW83** [TA2k/ioBroker.euhome#4]
Hallo,

in der App wird der prozentuelle Status des Zubehörs (zB Bürste) angezeigt. Jedoch im Adapter haben alle Datenpunkte den gleichen Wert. Ich gehe davon aus, dass dies Prozentwerte sind. Aber da diese alle gleich sind, und nicht einmal einer damit übereinstimmt, wollte ich fragen, wie man das korrigieren kann?

Danke & Lg, Michael

**Comment by TA2k** [comment]
und andere Objekte könnten nicht die Werte enthalten?

**Comment by MikeW83** [comment]
> und andere Objekte könnten nicht die Werte enthalten?

Nein, andere Objekte im ioBroker haben diese Informationen nicht. Es gibt für das Zubehör ja nur diese speziellen Datenpunkte.

**Comment by MikeW83** [comment]
Daten im ioBroker:
![Screenshot 2023-11-18 073123](https://github.com/TA2k/ioBroker.euhome/assets/35097389/fabd59f2-d557-48b7-b31f-592000a2a5c1)

und hier die App:
![Screenshot_20231118_072951_eufy Clean](https://github.com/TA2k/ioBroker.euhome/assets/35097389/a18ee4f3-d380-4f6a-83ad-2738976bac65)

**Comment by TA2k** [comment]
Und in den anderen DPS is auch nichts?

**Comment by MikeW83** [comment]
> Und in den anderen DPS is auch nichts?

![image](https://github.com/TA2k/ioBroker.euhome/assets/35097389/d8277e55-1391-4817-913e-eeed580e30f4)

**Comment by TA2k** [comment]
Auch erstmal keine Idee es werden eigentlich alle Daten abgefragt müsste ich nur mit Zugangsdaten reinschauen
tombox2020@gmail.com

**Comment by MikeW83** [comment]
> Auch erstmal keine Idee es werden eigentlich alle Daten abgefragt müsste ich nur mit Zugangsdaten reinschauen [tombox2020@gmail.com](mailto:tombox2020@gmail.com)

Kann ich hier irgendwie anders unterstützen? Gebe ungern Zugangsdaten weiter. :-(

**Comment by TA2k** [comment]
Leider nicht muss sehen was die app empfängt
